roles/edpm_nova/defaults/main.yml

```diff
@@ -31,3 +31,8 @@ edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compu
 
 # certs
 edpm_nova_tls_certs_enabled: "{{ edpm_tls_certs_enabled | default(False) }}"
+
+#Defaults for  PCI whitelist derive script, if nic-partitioning is enabled
+edpm_nova_sriov_derive_pci_whitelist: true
+#TBD: This can be an ENV variable
+edpm_nova_default_sriov_conf_file: 03-sriov-nova.conf
```

Comment:
@vcandapp is thi required? can we get the file name list from configmaps flattened for nova role?

Reply:
Will take this feature as part of feature pack 1